hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java

```diff
@@ -152,8 +151,12 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
     OMResponse.Builder omResponse = OmResponseUtil.getOMResponseBuilder(
         getOmRequest());
     OmBucketInfo omBucketInfo = null;
-    if (bucketInfo.getBucketLayout() == null || bucketInfo.getBucketLayout()
-        .equals(BucketLayoutProto.LEGACY)) {
+
+    // If the user has set a value, bucketInfo.hasBucketLayout() would be true,
```

Comment:
Please modify the comment to
```

    // bucketInfo.hasBucketLayout() would be true when user sets bucket layout.
    // Now, OM will create bucket with the user specified bucket layout.
    // When the value is not specified by the user, OM will use the
    // "ozone.default.bucket.layout" configured value.
```

Reply:
Thank you for the suggestions, I have incorporated the changes!